Maquetado/scss/details.scss

```diff
@@ -5,20 +5,32 @@
 }
 
 .book {
-  align-items: center;
   background-color: $white;
   box-shadow: 4px 4px 10px 0 $black-transparent;
   display: flex;
-  flex-flow: column wrap;
+  flex-wrap: wrap;
   height: 100%;
   margin: 0 auto;
-  max-height: 415px;
```

Comment:
Why you delete this?

Reply:
Because if i reduce the screen size and the info goes under the img, then the background doesn't fit. But it's about responsive stuff so i can undelete it.